src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEPending.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
+
+const COEPending = props => {
+  return (
+    <>
+      <va-alert status="warning">
+        <h3 slot="headline">
+          {props.pending
+            ? 'We need more information from you'
+            : 'We’re reviewing your application'}
+        </h3>
+        <p>You applied on:  June 30, 2020</p>
+        <div>
+          <p>
+            {props.pending
+              ? "You'll need to upload documents before we can make a decision on your COE application."
+              : "If you qualify for a Certificate of Eligibility, we'll notify you by email or mail to let you know how to get your COE."}
+          </p>
+          <a href="/housing-assistance/home-loans/apply-for-coe-form-26-1880/eligibility">
+            Go to your VA home loan COE page to see status details
+          </a>
+        </div>
+      </va-alert>
+      <div>
+        <h2>Should I apply again?</h2>
+        <p>
+          No. We’re reviewing your current application, and submitting a new
+          application won’t affect our decision or speed-up the process.
+        </p>
+        <p>
+          If more than 5 business days have passed since you submitted your
+          application and you haven’t heard back, please don’t apply again. Call
+          our toll-free number at <Telephone contact="8778273702" />.
+        </p>
+        <p>
+          We only recommend applying again if you’ve already worked with our VA
+          home loans case management team, and they’ve advised you to reapply.
+        </p>
+        <a href="#">Get instructions for downloading a VA.gov PDF</a>
+      </div>
+      <div className="vads-u-padding-top--4">
+        <a href="#">
+          <i className="fas fa-download vads-u-padding-right--1" /> Download
+          your COE (PDF) 0.20MB
+        </a>
+      </div>
+      <div>
+        <h2>What if my COE has errors?</h2>
+        <p>What if I need to make changes to my COE?</p>
+        <ul>
+          <li>
+            Make changes to your COE (correct an error or update your
+            information), <strong>or</strong>
+          </li>
+          <li>Apply for a restoration of entitlement</li>
+        </ul>
+      </div>
+    </>
+  );
```

Comment:
Looks like this block can be removed from the pending component according to the [mockups](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/141341802/simulate/sitemap)

Reply:
Same as above